docker-compose.client.yml

```diff
@@ -10,3 +10,14 @@ services:
     environment:
       - BALENACLOUD_API_KEY=${BALENACLOUD_API_KEY}
       - BALENACLOUD_ORG=${BALENACLOUD_ORG}
+  core:
+    build: ./core
+    network_mode: 'host'
+    volumes:
+      - 'core-storage:/data'
+      - 'reports-storage:/reports'
+      - /var/run/docker.sock:/var/run/docker.sock
+    environment:
+      - UDEV=0
+      - WORKER_PORT=${WORKER_PORT}
+      - CORE_PORT=${CORE_PORT}
```

Comment:
```suggestion
```
Do we need/use this value anymore?

Reply:
Looks like we have some stale values and defaults in here:
https://github.com/balena-os/leviathan/blob/ryan/core-to-client/core/config/default.js